src/test/java/core/basesyntax/service/impl/FileWriterServiceImplTest.java

```
@@ -0,0 +1,79 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FileWriterServiceImplTest {
+    private FileWriterService fileWriterService;
+
+    @BeforeEach
+    void setUp() {
+        fileWriterService = new FileWriterServiceImpl();
+    }
+
+    @Test
+    void write_validOutputDataAndFilePath_ok() {
+        String outputData = String.join(System.lineSeparator(), "banana,152",
+                "apple,90");
+        String fileName = "src/test/resources/reportToWrite.csv";
+        fileWriterService.write(outputData, fileName);
+        String expected = String.join(System.lineSeparator(), "fruit,quantity",
+                "banana,152", "apple,90");
+        String actual;
+        try {
+            actual = Files.readString(Path.of(fileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from file: " + fileName, e);
+        }
```

You can throw an exception in method signature